### DISCUSSION TITLE
The totalfetchrequests_total always rises periodically, even if no action is taken

### DISCUSSION THREAD

**Discussion by runkecheng** [strimzi/strimzi-kafka-operator#9319] [Q&A] [answered]
kafka_server_brokertopicmetrics_totalfetchrequests_total

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/68526782/6c6ca9c3-ecf8-40d7-8d75-89ee3cf43a3a)

Where can I view the capture logic for this indicator and why is this happening?

**Comment by scholzj** [comment, marked as answer]
Not sure what the metric exactly means. But if you have the Topic Operator enabled, or the Cruise Control, they are using some internal Kafka topics. So they might be fetching on them even when your applications are not running. Similarly, Kafka Exporter will read the consumer offsets as far as I know. Could that explain it?
